### ISSUE TITLE
Visualize not working for Sales Hub

### ISSUE THREAD

**Issue by bennyboy23** [scottdurow/NetworkView#21]
When you click on the visualize button, it does not show data when in the sales hub app in dynamics 365 v9. Is this a known issue?

**Comment by scottdurow** [comment]
Yes - there is an issue with the Unified Client that I need to work around - I have a fix which I’ll release soon.

**Comment by bennyboy23** [comment]
Thank you scott for the swift reply, looking forward to the fix!

**Comment by sebs211** [comment]
Hello Scott,
Do you have some news on this issue, please.
I install your solution for a customer, he really like it.
But he want to use it with the new HUB interface.
Regards,

**Comment by scottdurow** [comment]
Sorry about the delay in getting this out - I have been working on making a Custom
Control for it - but the framework isn’t released yet publicly. I think I’ll release the old web resource version in the mean time.

**Comment by sebs211** [comment]
Ok, thank you for your reactivity.
So we are waiting for your update :)

**Comment by sebs211** [comment]
hello Scott,
Any news about this subject ?

**Comment by nettellect** [comment]
Hi Scott,
any update of this one as it was expected to come out soon.

**Comment by RaghurajB** [comment]
@scottdurow do we have a fix for the issue which is blocking the visualization to load in the UCI?
